### PR TITLE
Add nullable annotations

### PR DESCRIPTION
**Pull request by dereuromark**
Fixes https://github.com/cakephp/bake/issues/469 and PHPStan problems with nullable DB fields on entity annotations.

### CODE REVIEW

**Comment by dereuromark** [comment]
I appreciate help here if someone has a bit of spare time.
I did https://github.com/dereuromark/cakephp-ide-helper/pull/95 already, but for now I dont have too much time to fix all those tests here in the bake plugin.

**Comment by dereuromark** [comment]
Should we only check the keys maybe? seems like the values can differ per DB type.
Or is this a bug/issue?

**Comment by smarek** [comment]
CI not passing here should be fixed by recent changes to .travis I think, is it possible to re-run the test here?

**Comment by dereuromark** [comment]
Those are fails due to my additions. But I am saying that we might not want to check the results so exactly, it could be fine to check only the keys of the array in some cases.

File: src/Shell/Task/ModelTask.php [hunk 1]
```diff
@@ -560,7 +560,9 @@ public function getEntityPropertySchema(Table $model)
 
             $properties[$column] = [
                 'kind' => 'column',
-            ] + $columnSchema;
+                'type' => $columnSchema['type'],
+                'null' => $columnSchema['null'],
+            ];
```
Comment (dereuromark) [inline src/Shell/Task/ModelTask.php]: Why not passing all?

Comment (markstory) [reply src/Shell/Task/ModelTask.php]: We don't need all the properties right now, and it creates a wider interface than we need with more chances to break behavior.

**Pull request merged**